proto/caikit_data_model_common_runtime.proto

```diff
@@ -18,7 +18,7 @@ message ModelInfo {
   string model_path = 1;
   string name = 2;
   int64 size = 3;
-  map<string, string> metadata = 4;
+  google.protobuf.Struct metadata = 4;
```

Comment:
looks like an api-breaking change, but this is the first time we'll be releasing a server implementation here that actually returns this message...

It's probably fine? But might be slightly problematic if somebody calls the server with outdated protos. 🤷

Reply:
So, I think the product team was not using this API prior to this change, but I can confirm if this will impact other users!

Reply:
@joerunde , running `RUNTIME_LIBRARY=caikit_nlp python -m caikit.runtime.dump_services grpc_interface_dir` creates this file where google.protobuf.Struct is used:
```
$ cat grpc_interface_dir/caikit_data_model.common.runtime.modelinfo.proto

/*------------------------------------------------------------------------------
 * AUTO GENERATED
 *----------------------------------------------------------------------------*/

syntax = "proto3";
package caikit_data_model.common.runtime;
import "google/protobuf/struct.proto";


/*-- MESSAGES ----------------------------------------------------------------*/

message ModelInfo {

  /*-- nested messages --*/

  /*-- fields --*/
  string model_path = 1;
  string name = 2;
  int64 size = 3;
  google.protobuf.Struct metadata = 4;
  bool loaded = 7;
  string module_id = 5;
  map<string, string> module_metadata = 6;
}
```
So it looks like this is an upstream change that is being reflected here.

Reply:
Yup, I know it's an upstream change, I was just wondering if we knew whether or not it would impact anybody